test/test_default_readtable.js

```diff
@@ -59,7 +61,7 @@ test('should parse keywords', t => {
     });
   }
 
-  testParseKeyword('await ', 'await');
+  // testParseKeyword('await ', 'await'); TODO: uncomment when await is a keyword
```

Comment:
It's not a keyword in ES2015-16. Test fails at https://github.com/sweet-js/sweet.js/blob/22e5d182eb764d8427a7f447a69ad08ab777cb7a/test/expressions/identifier-expression.js#L31.
I could comment out that test and add `async` as well, but I figured I'd wait until we were using `babel-preset-es2017`. That would be after an upgrade to the Shift suite I guess.